checkov/sca_package/scanner.py

```diff
@@ -2,103 +2,110 @@
 import json
 import logging
 import os
+import time
 from collections.abc import Iterable, Sequence
 from pathlib import Path
-from typing import Tuple, Dict, Any
+from typing import Dict, Any
 
+import requests
 from aiomultiprocess import Pool
 
-from checkov.common.bridgecrew.vulnerability_scanning.integrations.package_scanning import package_scanning_integration
-
-TWISTCLI_FILE_NAME = 'twistcli'
+from checkov.common.bridgecrew.platform_integration import bc_integration
+from checkov.common.util.data_structures_utils import merge_dicts
+from checkov.common.util.file_utils import compress_file_gzip_base64, decompress_file_gzip_base64
+from checkov.common.util.http_utils import get_default_get_headers, get_default_post_headers
 
 
 class Scanner:
     def __init__(self) -> None:
-        self.twistcli_path = Path(TWISTCLI_FILE_NAME)
-
-    def setup_twictcli(self) -> None:
-        try:
-            if not self.twistcli_path.exists():
-                package_scanning_integration.download_twistcli(self.twistcli_path)
-        except Exception:
-            logging.error("Failed to setup twictcli for package scanning", exc_info=True)
-            raise
-
-    def cleanup_twictcli(self) -> None:
-        if self.twistcli_path.exists():
-            self.twistcli_path.unlink()
-        logging.info('twistcli file removed')
-
-    def scan(self, input_output_paths: "Iterable[Tuple[Path, Path]]", cleanup_twictcli: bool = True) \
-            -> "Sequence[Dict[str, Any]]":
-        self.setup_twictcli()
+        self.base_url = bc_integration.api_url
+        self.headers = merge_dicts(
+            get_default_get_headers(bc_integration.bc_source, bc_integration.bc_source_version),
+            {"Authorization": bc_integration.get_auth_token()},
```

Comment:
@Eliran-Turgeman This might be a time-limited token, and so should be computed as close to the request as possible. It contains a cache by itself, so don't worry calling it too many times :)